code/Control/UserDefinedFormController.php

```diff
@@ -105,7 +106,7 @@ protected function addUserFormsValidatei18n()
      *
      * @return array
      */
-    public function index()
+    public function index(HTTPRequest $request)
```

Comment:
`public function index(HTTPRequest $request = null)` please so we can maintain backwards compatibility